src/platforms/esp32/components/avm_builtins/i2c_driver.c

```diff
@@ -86,8 +86,15 @@ struct I2CData
 {
     i2c_cmd_handle_t cmd;
     term transmitting_pid;
+    i2c_port_t i2c_num;
 };
 
+#define VALIDATE_NOT_INVALID(moniker)                       \
```

Comment:
If you like with can prefix this, like I2C_VALIDATE_NOT_INVALID or I2C_INIT_VALIDATE_NOT_INVALID

Reply:
Good point.  Done.